blog/tests/models_test.py

```diff
@@ -0,0 +1,64 @@
+"""Tests for model's Blog app."""
```

Comment:
What are the tests for the blog app of the model?

Reply:
```
"""Tests for models of Blog app."""
```
English error? Now I think good.